docs/source/testing/graphql-playground.mdx

```diff
@@ -20,51 +20,67 @@ To try out dev graphs:
+[GraphQL Playground](https://github.com/graphql/graphql-playground) is a graphical, interactive, in-browser GraphQL IDE, created by [Prisma](https://www.prisma.io/) and based on [GraphiQL](https://github.com/graphql/graphiql).
 
-In development, Apollo Server enables GraphQL Playground on the same URL as the GraphQL server itself (e.g. `http://localhost:4000/graphql`) and automatically serves the GUI to web browsers.  When `NODE_ENV` is set to `production`, GraphQL Playground (as well as introspection) is disabled as a production best-practice.
+In development, Apollo Server enables GraphQL Playground on the same URL as the GraphQL server itself (e.g. `http://localhost:4000/graphql`) and automatically serves the GUI to web browsers.  When `NODE_ENV` is set to `production`, GraphQL Playground (as well as introspection) is disabled as a production best-practice. It does this by installing the `ApolloServerPluginUIGraphQLPlayground` plugin with default settings.
 
 ![GraphQL Playground](../images/graphql-playground.png)
 
 ## Configuring Playground
 
-The Apollo Server constructor contains the ability to configure GraphQL Playground with the `playground` configuration option. The options can be found on GraphQL Playground's [documentation](https://github.com/prismagraphql/graphql-playground/#usage).
+If you'd like to configure playground, you can add `ApolloServerPluginUIGraphQLPlayground` to your Apollo Server yourself. contains the ability to configure GraphQL Playground with the `playground` configuration option. The options can be found on GraphQL Playground's [documentation](https://github.com/graphql/graphql-playground/#usage).
 
 ```js
+const { ApolloServer } = require('apollo-server');
+const { ApolloServerPluginUIGraphQLPlayground } = require('apollo-server-core');
+
 new ApolloServer({
-typeDefs,
-resolvers,
-playground: {
-  settings: {
-    'editor.theme': 'light',
-  },
-  tabs: [
-    {
-      endpoint,
-      query: defaultQuery,
-    },
+  typeDefs,
+  resolvers,
+  plugins: [
+    ApolloServerPluginUIGraphQLPlayground({
+      settings: {
+        'editor.theme': 'light',
+      },
+      tabs: [
+        {
+          endpoint,
+          query: defaultQuery,
+        },
+      ],
+    }),
   ],
-},
 });
 ```
 
-## Enabling GraphQL Playground in production
+## Disabling Playground in development
 
-To enable GraphQL Playground in production, introspection and the playground can be enabled explicitly in the following manner.
+If you don't want to run GraphQL Playground at all, you have two choices. First, you can replace it with a different UI by installing a plugin that implements the [`renderUIPage` event](../integrations/plugins/#renderuipage). Alternatively, you can install the `ApolloServerPluginUIDisabled` plugin which stops Apollo Server from serving any HTML UI.
```

Comment:
I was going based on how plugins.md mostly uses the word events rather than hooks (except for end hooks) but in this context, it definitely looks better as hook.